### DISCUSSION TITLE
Enhancement: Change the keybind to mute the game to another key.

### DISCUSSION THREAD

**Discussion by Eliptical** [FunkinCrew/Funkin#1616] [Ideas]
I am only requesting this enhancement as I use arrow keys and sometimes, if you aren't paying enough attention. You can be trying to hit a lot of notes and then your pinky slips down and hits the mute key which can mess you up hard. 
I am assuming it's meant to be there for quick and easy convivence. So that's why I think the Num-Pad Enter key would work best as the default bind, it's still quick to access while playing while not being in the way of one of the control schemes.

**Comment by GStudiosX2** [comment]
Volume button I don't think is defined in the game itself it's from haxeflixel so there's no fix to that

**Comment by dinkley** [comment]
> Volume button I don't think is defined in the game itself it's from haxeflixel so there's no fix to that

You can indeed change it!
`FlxG.sound.muteKeys = ["M"];`

Same applies for:
`FlxG.sound.volumeUpKeys
`
and
`FlxG.sound.volumeUpKeys
`

**Comment by GStudiosX2** [comment]
> > Volume button I don't think is defined in the game itself it's from haxeflixel so there's no fix to that
> 
> You can indeed change it!
> `FlxG.sound.muteKeys = ["M"];`
> 
> Same applies for:
> `FlxG.sound.volumeUpKeys `
> and
> `FlxG.sound.volumeUpKeys `

i mean why would someone add it just cuz one random person wants it tho lol its like a tiny change too and it isnt even a big deal
